src/scalper.ts

```diff
@@ -601,32 +601,74 @@ export class Scalper {
     const stdDevSpread = this.impliedVol / Math.sqrt((365 * 24 * 60 * 60) / scalperWindow) * zScore;
     const slippageTolerance = Math.min(stdDevSpread/ 2, slippageMax.get(this.symbol));
     const deltaThreshold = Math.max(dipTotalGamma * stdDevSpread * fairValue * gammaThreshold, this.minSpotSize);
+    let hedgePrice = hedgeDeltaTotal < 0 ? fairValue * (1 + slippageTolerance) : fairValue * (1 - slippageTolerance);
     if (Math.abs(hedgeDeltaTotal * fairValue) < (deltaThreshold * fairValue)) {
       console.log(this.symbol, "Delta Netural <", deltaThreshold);
       return;
     } else {
-      console.log(this.symbol, "Above delta threshold:", hedgeDeltaTotal, ">", deltaThreshold);
+      // Adjust delta for slippage allowed
+      const slipapgeDelta = getDIPDelta(dipProduct, hedgePrice, this.symbol);
+      const dipDeltaDiff = slipapgeDelta - dipTotalDelta;
+      hedgeDeltaTotal = hedgeDeltaTotal + dipDeltaDiff;
+      console.log(this.symbol, "Adjust Slippage Delta by", dipDeltaDiff, "to", hedgeDeltaTotal);
+      if (Math.abs(hedgeDeltaTotal * fairValue) < (deltaThreshold * fairValue)) {
+        console.log(this.symbol, "Delta Netural: Slippage <", deltaThreshold);
+        return;
+      }
+      console.log(this.symbol, "Outside delta threshold:", Math.abs(hedgeDeltaTotal), "vs.", deltaThreshold);
     }
-  
-    // Place an order to get delta neutral.
-    const hedgePrice = hedgeDeltaTotal < 0 ? fairValue * (1 + slippageTolerance) : fairValue * (1 - slippageTolerance);
     
-    // Splice Order depending on book depth
+    // Check order book depth
```

Comment:
are we checking depth or just top of book?

Reply:
in liquidityCheckSplice, repositioned